cdk/README.md

```diff
@@ -36,17 +36,23 @@ $ pip install -r requirements.txt
 
 You must now ensure that you are signed into AWS CLI:
 
-Instructions for configuring AWS CLI: 
-- https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+Instructions for configuring AWS CLI:
 
-Sign in instructions for AWS Educate accounts:
-- Navigate to awseducate.com and sign in
-- Go to "My Classrooms" and select go to classroom for the class' account you wish to deploy to
-- Select "Account Details" and "Show" next to AWS CLI
-- Copy and paste the block code that is shown into ~/.aws/credentials
+-   https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+
+Sign in instructions for AWS accounts:
+
+-   Navigate to https://clemson-makerspace.awsapps.com/start/ and sign in
+-   Click on "AWS Account" and select the tab for your dev account. It should say "dev-{your-clemson-username}"
+-   Select "Command line or programmatic access" on the row for "AdministratorAccess"
+-   Copy and paste the block code that is shown into ~/.aws/credentials
```

Comment:
Sounds good! I opened a new PR because this pointed to a fork that existed before the repo was made open-source. The updated readme in #53 should address these concerns.